### PR TITLE
Change tooltips to use Tippy instead

### PR DESCRIPTION
**Pull request by L-Sva**
Implements #51.

This update removes the title tag and instead displays the same information in using Tippy. This was only implemented for the search results.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,6 +5,7 @@
   "dependencies": {
     "@remix-run/cloudflare-workers": "^1.6.8",
     "@remix-run/react": "^1.6.8",
+    "@tippyjs/react": "^4.2.6",
```
Comment (Aprillion) [inline package.json:8]: please also commit changes to `package-lock.json` after running `npm i`

**Comment by ccstan99** [comment]
Maybe I missed a discussion somewhere... Could you please explain what is the goal of this enhancement? Running this on my local dev environment makes the tensorflow score result much more prominent than necessary and isn't lined up with the actual question score either. So I'm not sure this is a desirable direction. Thoughts @plexish?
Screenshot 2023-02-21 at 12 26 42 PM copy

**Review by ccstan99 — COMMENTED** [review]
Maybe I missed a discussion somewhere... Could you please explain what is the goal of this enhancement? Running this on my local dev environment makes the tensorflow score result much more prominent than necessary and isn't lined up with the actual question score either. So I'm not sure this is a desirable direction. Thoughts @plexish?
Screenshot 2023-02-21 at 12 26 42 PM copy

**Comment by plexish** [comment]
@ccstan99 this was meant to deal with #51, an issue you opened a long time ago. Might have been misunderstood though?

**Comment by ccstan99** [comment]
Oh, I see. It looks like the other tooltips had already been fixed previously or at least replaced by text under icons. This PR only adds Tippy to the search results, which I think both the color & behavior unnecessarily draws attention to those scores. Nevertheless, we can close #51 anyway. Thanks!

**Comment by ccstan99** [comment]
I just looked at the original issue #51  and @Aprillion & @ChrisRimmer discussing the implications on screen readers. Was Tippy supposed to resolve that? Sorry, if I closed this out prematurely. If it's only for aesthetic reasons as I originally wrote it for, then we're all set. Otherwise, this can be reopened. Thanks again!

**Pull request closed without merging**